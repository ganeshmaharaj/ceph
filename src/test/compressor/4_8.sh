#!/bin/sh

#"8K/4/8"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 8 --bluestore_compression_min_blob_size 8192 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"16K/4/8"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 8 --bluestore_compression_min_blob_size 16384 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"64K/4/8"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 8 --bluestore_compression_min_blob_size 65536 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"256K/4/8"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 8 --bluestore_compression_min_blob_size 262144 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

#"4M/4/8"
CEPH_ARGS="--bluestore_max_ops 4 --async_compressor_threads 8 --bluestore_compression_min_blob_size 4194304 --conf ./ceph.conf" bin/unittest_compression --gtest_filter=PerfTest*

