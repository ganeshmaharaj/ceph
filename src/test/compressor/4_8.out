Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/8192 in 1288 msec. cores/threads = 4/8
ctx switches 435, use(%) 94, user(%) 92, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (4617 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 10287mb ratio=0.449576 via snappy/8192 in 14616 msec. cores/threads = 4/8
ctx switches 4508, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_snappy (17901 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 9458mb ratio=0.41334 via zlib+isal/8192 in 23916 msec. cores/threads = 4/8
ctx switches 9450, use(%) 100, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (27173 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 7512mb ratio=0.328286 via zlib/8192 in 91619 msec. cores/threads = 4/8
ctx switches 27085, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_zlib (94876 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 8086mb ratio=0.353361 via zlib+qat/8192 in 66830 msec. cores/threads = 4/8
ctx switches 7355199, use(%) 59, user(%) 24, kernel(%) 34
[       OK ] PerfTest.perf_compare_qat (73012 ms)
[----------] 5 tests from PerfTest (217579 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (217579 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/16384 in 1190 msec. cores/threads = 4/8
ctx switches 463, use(%) 93, user(%) 92, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (4425 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9731mb ratio=0.425288 via snappy/16384 in 14357 msec. cores/threads = 4/8
ctx switches 4274, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_snappy (17528 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8862mb ratio=0.387299 via zlib+isal/16384 in 20405 msec. cores/threads = 4/8
ctx switches 6070, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (23571 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 7102mb ratio=0.3104 via zlib/16384 in 76379 msec. cores/threads = 4/8
ctx switches 22283, use(%) 100, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_zlib (79542 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7774mb ratio=0.339746 via zlib+qat/16384 in 39457 msec. cores/threads = 4/8
ctx switches 3899917, use(%) 46, user(%) 20, kernel(%) 26
[       OK ] PerfTest.perf_compare_qat (45007 ms)
[----------] 5 tests from PerfTest (170073 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (170073 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/65536 in 998 msec. cores/threads = 4/8
ctx switches 344, use(%) 98, user(%) 97, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (4141 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9044mb ratio=0.395235 via snappy/65536 in 15797 msec. cores/threads = 4/8
ctx switches 4856, use(%) 99, user(%) 91, kernel(%) 8
[       OK ] PerfTest.perf_compare_snappy (18856 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8421mb ratio=0.36801 via zlib+isal/65536 in 18610 msec. cores/threads = 4/8
ctx switches 5512, use(%) 99, user(%) 98, kernel(%) 1
[       OK ] PerfTest.perf_compare_isal (21682 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 6675mb ratio=0.29174 via zlib/65536 in 79668 msec. cores/threads = 4/8
ctx switches 24074, use(%) 99, user(%) 93, kernel(%) 5
[       OK ] PerfTest.perf_compare_zlib (82735 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7484mb ratio=0.327073 via zlib+qat/65536 in 21334 msec. cores/threads = 4/8
ctx switches 1281622, use(%) 45, user(%) 26, kernel(%) 17
[       OK ] PerfTest.perf_compare_qat (27005 ms)
[----------] 5 tests from PerfTest (154419 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (154419 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/262144 in 1206 msec. cores/threads = 4/8
ctx switches 571, use(%) 97, user(%) 95, kernel(%) 1
[       OK ] PerfTest.perf_compare_dummy (4359 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9043mb ratio=0.395201 via snappy/262144 in 36620 msec. cores/threads = 4/8
ctx switches 1174666, use(%) 61, user(%) 46, kernel(%) 15
[       OK ] PerfTest.perf_compare_snappy (39682 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8310mb ratio=0.363192 via zlib+isal/262144 in 17453 msec. cores/threads = 4/8
ctx switches 5181, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (20557 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 6564mb ratio=0.286855 via zlib/262144 in 79652 msec. cores/threads = 4/8
ctx switches 23557, use(%) 99, user(%) 96, kernel(%) 3
[       OK ] PerfTest.perf_compare_zlib (82704 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7420mb ratio=0.324286 via zlib+qat/262144 in 22917 msec. cores/threads = 4/8
ctx switches 1084637, use(%) 28, user(%) 21, kernel(%) 6
[       OK ] PerfTest.perf_compare_qat (29005 ms)
[----------] 5 tests from PerfTest (176307 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (176307 ms total)
[  PASSED  ] 5 tests.
Note: Google Test filter = PerfTest*
[==========] Running 5 tests from 1 test case.
[----------] Global test environment set-up.
[----------] 5 tests from PerfTest
[ RUN      ] PerfTest.perf_compare_dummy
Summary: orig 22883mb -> 22883mb ratio=1 via none/4194304 in 2876 msec. cores/threads = 4/8
ctx switches 1173, use(%) 99, user(%) 98, kernel(%) 0
[       OK ] PerfTest.perf_compare_dummy (6012 ms)
[ RUN      ] PerfTest.perf_compare_snappy
Summary: orig 22883mb -> 9043mb ratio=0.39519 via snappy/4194304 in 29708 msec. cores/threads = 4/8
ctx switches 62762, use(%) 67, user(%) 63, kernel(%) 4
[       OK ] PerfTest.perf_compare_snappy (32737 ms)
[ RUN      ] PerfTest.perf_compare_isal
Summary: orig 22883mb -> 8276mb ratio=0.361688 via zlib+isal/4194304 in 18590 msec. cores/threads = 4/8
ctx switches 5560, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_isal (21673 ms)
[ RUN      ] PerfTest.perf_compare_zlib
Summary: orig 22883mb -> 6528mb ratio=0.285316 via zlib/4194304 in 78794 msec. cores/threads = 4/8
ctx switches 23934, use(%) 99, user(%) 99, kernel(%) 0
[       OK ] PerfTest.perf_compare_zlib (81818 ms)
[ RUN      ] PerfTest.perf_compare_qat
Summary: orig 22883mb -> 7399mb ratio=0.323362 via zlib+qat/4194304 in 23274 msec. cores/threads = 4/8
ctx switches 993090, use(%) 32, user(%) 27, kernel(%) 2
[       OK ] PerfTest.perf_compare_qat (29004 ms)
[----------] 5 tests from PerfTest (171244 ms total)

[----------] Global test environment tear-down
[==========] 5 tests from 1 test case ran. (171244 ms total)
[  PASSED  ] 5 tests.
